mmcv/runner/hooks/sampler_seed.py

```diff
@@ -6,4 +6,4 @@
 class DistSamplerSeedHook(Hook):
 
     def before_epoch(self, runner):
-        runner.data_loader.sampler.set_epoch(runner.epoch)
+        runner.data_loader.batch_sampler.sampler.set_epoch(runner.epoch)
```

Comment:
This line of code will change the original behavior, i.e., the sampler will not be set for each epoch as expected.
The reason for bug #472 is that when the samplers implemented by Pytorch (e.g., `SequentialSampler`) does not have the function `set_epoch`.

Reply:
I got it. Thanks.